dist/messages/messages.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
It seems that this file currently won't be generated in or copied into dist/messages folder by default.